src/utils/apiWrapper.js

```diff
@@ -89,11 +91,18 @@ function convertToApiMap(map) {
 
 // TODO: api url should be /pois
 function getPOIs() {
-  return createRequest(REQUEST_METHODS.get, 'poi')
+  return createRequest(REQUEST_METHODS.get, 'pois')
     .then(res => res.data)
     .then(res => res.map(r => r.data).map(convertFromApiPOI))
 }
 
+function getPOIsByYear(year) {
+  // will soon be `maps/year/<year>`
+  return createRequest(REQUEST_METHODS.get, `pois/year/${year}`)
+    .then(res => res.data.pois)
+    .then(res => res.map(convertFromApiPOI))
```

Comment:
Something to look out for: the response for this might be a different structure than for pois